cg/server/admin.py

```diff
@@ -323,7 +324,15 @@ class PoolView(BaseView):
 class SampleView(BaseView):
     """Admin view for Model.Sample"""
 
-    column_exclude_list = ["invoiced_at"]
+    column_exclude_list = [
+        "age_at_sampling",
+        "from_sample",
+        "invoiced_at",
+        "loqusdb_id",
```

Comment:
I would like to keep this one:  "loqusdb_id"

Reply:
Is it OK to have it available via CLI command `cg export sample` or is it needed on an everyday basis?

Reply:
I re-added it